.travis.yml

```diff
@@ -1,6 +1,6 @@
+dist: xenial
+
 language: node_js
-node_js:
-  - "8"
 
 install:
   - npm install
```

Comment:
@mukkachaitanya The npm gets installed and managed via `nvm`.

Reply:
So, sir, I need that parameter?

Reply:
What I mean is that the `npm install` command may not be required at all.

Reply:
Ok. Understood sir.